client/lib/catch-js-errors/index.js

```diff
@@ -119,17 +121,24 @@ export default class ErrorLogger {
 	}
 
 	sendToApi( error ) {
-		const body = new window.FormData();
-		body.append( 'error', JSON.stringify( error ) );
+		const onUnableToRecordError = () => {
+			// eslint-disable-next-line no-console
+			console.error( 'Error: Unable to record the error in Logstash.' );
+		};
 
 		try {
-			window.fetch( 'https://public-api.wordpress.com/rest/v1.1/js-error', {
-				method: 'POST',
-				body,
-			} );
+			wpcom.req.post(
```

Comment:
What would happen when the user is not logged in? (for example, errors in the login screen)

Reply:
I've just manually tested this on the login screen and it works. 

This function is also used in the code for other logged-out features like verification emails.